.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Here's a :candy: while we look into your'' first issue'
```

Comment:
I think the apostrophe in `Here's` needs to be escaped with a second single quote, and there shouldn't be two single quotes after `your`. Colons also have tricky escape sequences:

https://yaml.org/spec/1.2/spec.html#id2788097

Perhaps we should go for a more straightforward

> 'Thanks for submitting your first issue to Wagtail!'

and 

> 'Thanks for submitting your first PR to Wagtail!'

Reply:
Looks good! This is my first time working with yaml files and I did try out these messages on my own repos but wasn't sure what was wrong with the message that the :candy: didn't appear (which was sad)

Making the changes as suggested by you! :)